backend/app/Models/Friend.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Friend extends Model
+{
+    //
+    public static function areFriends($user_id1, $user_id2)
```

Comment:
this might make more sense as a non-static function e.g.
 ```
public function areFriends($targetUser) {
then compare: $this->id and $targetUser
}
```

being able to call `$user1->areFriends($user2)` just feels cleaner (and less verbose) than calling it statically IMO